test/JDBC/input/operator_binary_whitespace-vu-prepare.sql

```diff
@@ -0,0 +1,12 @@
+create view v1_bitop as select 99&+2 as c1, 88|-9 as c2, 77^~5 as c3
+go
+create procedure p1_bitop  as select 99&~5|-4

```

Comment:
Would you add a test case that put multiple expressions into procedure declare like : 
```
select 99&+2 as c1, 88|-9 as c2, 77^~5,a,b,c from test_table
```

Reply:
testcases added